src/scales/scale.ts

```diff
@@ -61,6 +61,11 @@ namespace Plottable.Scales {
      * `scaleTransformation`.
      */
     getTransformationDomain(): [number, number];
+
+    /**
+     * Returns value in *Transformation Space* for the provided *screen space*.
+     */
+    invertTransformation(value: number): number;
```

Comment:
nit: call this `inverted` to signify this applies the already-inverted transformation (rather than saying this method inverts the transformation itself)